embassy-boot/src/boot_loader.rs

```diff
@@ -49,16 +49,20 @@ pub struct BootLoaderConfig<ACTIVE, DFU, STATE> {
     pub state: STATE,
 }
 
-impl<'a, FLASH: NorFlash>
+impl<'a, ActiveFlash: NorFlash, DFUFlash: NorFlash, StateFlash: NorFlash>
```

Comment:
Could you change the names ACTIVE, DFU and STATE which is also used elsewhere in the code.

Reply:
Done in af2b4df.